js/reference.js

```diff
@@ -2015,6 +2015,24 @@
             return this._display;
         },
 
+        /**
+         * The path to the table where the favorite terms are stored. Checks the
+         * 2021:table-config annotation for `user_favorites.storage_table`. If
+         * present, returns a string in the form of:
+         *   "/ermrest/catalog/<catalog-id>/entity/<schema>:<table>".
+         *
+         * If the annotation was not properly defined or not defined at all, this
+         * will return null.
+         */
+        get favoritesPath () {
```

Comment:
Let's remove this one and change the `_favoritesPath` that you added in table to be `favoritesPath`. There's no reason to add an API for this in Reference when you can just directly get it from the table